openslides_backend/main.py

```diff
@@ -45,13 +47,28 @@ def __init__(self, view_name: str, *args: Any, **kwargs: Any) -> None:
 
         super().__init__(*args, **kwargs)
 
+    def loglevel(self) -> str:
+        if is_dev_mode():
+            return "DEBUG"
+        lvl = os.environ.get("OPENSLIDES_LOGLEVEL", "INFO").upper()
+        if lvl not in ("DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"):
+            raise ValueError("Invalid OPENSLIDES_LOGLEVEL: {lvl}")
+        return lvl
```

Comment:
I think this belongs in `openslides_backend/shared/env.py`

Reply:
Refactord